### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by 6hundred9**
Made it a draft for probably no reason at all

### CODE REVIEW

**Comment by 6hundred9** [comment]
alr

**Comment by 6hundred9** [comment]
you can just ping me in discord yk

File: RueI/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# RueI
+RueI is a **hint framework** for scp:sl. it provides the ability to show multiple hints at once, along with a lot of other features
+
+### For Server Owners
+Installing RueI is easy. download the dll, and move it into your *dependencies* folder for the plugin framework you are using. note that RueI requires Harmony 2.2.2, you can get that [here](https://github.com/pardeike/Harmony).
+
+### For Developers
+If you wish to start making plugins using RueI, most of the documentation will be here and in [the RueI site](https://ruemena.github.io/RueI/), if you have any questions you can ask it in [the EXILED server](https://discord.gg/4EQCJybWzz)
```
Comment (Misfiy) [inline RueI/index.md:12]: This leads to the same page

**Review by Misfiy — CHANGES_REQUESTED** [review]

File: RueI/index.md (hunk 1 above)
Comment (6hundred9) [reply RueI/index.md:12]: oh

**Comment by 6hundred9** [comment]
@Misfiy tell me stuff to add to docs

**Pull request merged**